wg-manifests/charter.md

```diff
@@ -1,70 +1,50 @@
 # WG Manifests Charter
 
+
 This charter adheres to the conventions, roles and organization management
 outlined in [wg-governance].
 
 ## Scope
 
-- Provide a catalog (centralized repository) of Kubeflow application manifests.
-- Provide a catalog of third-party apps for common services.
+We simply (automatically) synchronize the application and dependencies manifests to then elaborately combine (configure)them for full platform experience.
+Providing a consistent and tested end-to-end multi-tenant experience is the most important task of the platform/manifests WG.
+To achieve this we maintain an extensive testing suite that covers most basic scenarios users would expect from a Platform for ML orchestration.
+We also provide the documentation regarding, but not limited to installation, extension, security and architecture to enable users to run their own ML Platform on Kubernetes.
+Users may choose to derive from platform/manifests to create so called distributions, which are opinionated to satisfy individual requirements.
+Users may also choose to install individual components without the benefits of the platform.
```

Comment:
Can we format this as a list of bullet points, and combine the ones which are the the same idea?

This makes it easier to have discussions about each specific element of the scope, as some new elements are being proposed.